src/js/video.js

```diff
@@ -749,5 +749,7 @@ videojs.dom = Dom;
  */
 videojs.url = Url;
 
+videojs.TERMINATOR = TERMINATOR;
```

Comment:
If we have access to `Object.defineProperty` we should probably use it here to prevent overwriting this value. Others may think that's overkill, though.

Reply:
Seems reasonable, I'll do that